### DISCUSSION TITLE
Diesel barrels and Refilling them.

### DISCUSSION THREAD

**Discussion by WrldMachine** [PeasantFoundry/Mechtrauma#83] [Ideas]
i know that making diesel requires an very specific point of an certain process to produce it, yet im wondering if my campaign is doomed to fail since im running an diesel only submarine. not sure how many places will sell barrels or how many there are goin to be but an idea would be to able to refill diesel barrels, either by an recipe or by an crazy chemistry thing;

**Comment by PeasantFoundry** [comment]
A biofuel / hydrocarbon fuel machine is on the roadmap. The idea is to allow you to make your own fuel out of organic materials or some minerals.
